### DISCUSSION TITLE
Sdrpp for android

### DISCUSSION THREAD

**Discussion by JaxTrains** [AlexandreRouma/SDRPlusPlus#1397] [General]
I can not find a way to download sdrpp for android on this website

**Comment by KubaPro010** [comment]
its literally in the releases ([link](https://github.com/AlexandreRouma/SDRPlusPlus/releases/tag/nightly)) and if you're even lazier: https://github.com/AlexandreRouma/SDRPlusPlus/releases/download/nightly/sdrpp.apk

**Reply by JaxTrains** [reply]
Does not work.  Says this program is for older android tablets
Cgiera@comcast.net

**Reply by KubaPro010** [reply]
this you must have a weird phone/device, on my android 13 phone it installs with no problem

**Reply by JaxTrains** [reply]
Also, getting a message that the program is harmfull to my tablet and refuses to install the program.

**Reply by JaxTrains** [reply]
Finally got around the issue for installing SDRpp on my Samsung S6 Lite.  I have a new problem the display is too big for the screen.  I can't see all the features.  Is the any way to make the display smaller?

**Reply by KubaPro010** [reply]
By default SDR++ on android has its gui 2 times bigger than normal, if you can go to the display menu then change `DPI Scaling` from 200 to 100
